rules/configuration.smk

```diff
@@ -40,12 +40,14 @@ if 'condition' in samples:
                 f' condition {condition} and assembly {assembly}'
 
 
-# cut off trailing slashes and make absolute path
+# make absolute paths, cut off trailing slashes and nestle default directories in result_dir
```

Comment:
Isn't it nest instead of nestled? :angel:

Reply:
oops